packages/design-system/src/components/ButtonGroup/ButtonGroup.js

```diff
@@ -58,7 +58,7 @@ class ButtonGroup extends React.Component {
 }
 
 ButtonGroup.defaultProps = {
-  onIndexChange: () => {}
+  onIndexChange: (index, event) => {}
```

Comment:
It is actually the same: still `noop` function but now you have there not used arguments which could be bad in case of some linting rule in future.

Reply:
deleted this noop, and added check in call b1005187a0d51f7e88f2f764c0701d21ca564965